src/sentry/rules/base.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         project: Project,
         data: dict[str, Any] | None = None,
         rule: Rule | None = None,
+        **kwargs: Any,
```

Comment:
had to add this for mypy

Reply:
yes, because `SlackNotifyServiceAction` actually inherits from `RuleBase` :/